ebl-issuance/src/main/resources/standards/eblissuance/messages/eblissuance-v3.0.0-request.json

```diff
@@ -102,65 +181,26 @@
-          "equipmentReference": "NARU3472484"
-        }
-      }
-    ],
-    "isShippedOnBoardType": true,
-    "transportDocumentStatus": "ISSUED",
-    "cargoMovementTypeAtOrigin": "FCL",
-    "cargoMovementTypeAtDestination": "FCL",
-    "receiptTypeAtOrigin": "CY",
-    "deliveryTypeAtDestination": "CY",
-    "shippedOnBoardDate": "2023-12-19",
-    "termsAndConditions": "You agree that this transport document exist is name only for the sake of\ntesting your conformance with the DCSA EBL API. This transport document is NOT backed\nby a real shipment with ANY carrier and NONE of the requested services will be\ncarried out in real life.\n\nUnless required by applicable law or agreed to in writing, DCSA provides\nthis JSON data on an \"AS IS\" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF\nANY KIND, either express or implied, including, without limitation, any\nwarranties or conditions of TITLE, NON-INFRINGEMENT, MERCHANTABILITY,\nor FITNESS FOR A PARTICULAR PURPOSE. You are solely responsible for\ndetermining the appropriateness of using or redistributing this JSON\ndata and assume any risks associated with Your usage of this data.\n\nIn no event and under no legal theory, whether in tort (including negligence),\ncontract, or otherwise, unless required by applicable law (such as deliberate\nand grossly negligent acts) or agreed to in writing, shall DCSA be liable to\nYou for damages, including any direct, indirect, special, incidental, or\nconsequential damages of any character arising as a result of this terms or conditions\nor out of the use or inability to use the provided JSON data (including but not limited\nto damages for loss of goodwill, work stoppage, computer failure or malfunction, or any\nand all other commercial damages or losses), even if DCSA has been advised of the\npossibility of such damages.\n",
-    "carrierCode": "HMM",
-    "carrierCodeListProvider": "SMDG",
-    "charges": [
-      {
-        "chargeName": "Fictive transport document fee",
-        "currencyAmount": 1.0,
-        "currencyCode": "EUR",
-        "paymentTermCode": "COL",
-        "calculationBasis": "Per transport document",
-        "unitPrice": 1.0,
-        "quantity": 1
+        ]
       }
-    ],
-    "transports": {
-      "plannedDepartureDate": "2023-12-19",
-      "plannedArrivalDate": "2023-12-21",
-      "portOfLoading": {
-        "UNLocationCode": "DKAAR"
-      },
-      "portOfDischarge": {
-        "UNLocationCode": "DEBRV"
-      },
-      "vesselVoyages": [
-        {
-          "vesselName": "HMM Algeciras",
-          "carrierExportVoyageNumber": "402E"
-        }
-      ]
-    }
+    ]
   },
   "issueTo": {
-    "sendToPlatform": "SEND_TO_PLATFORM_PLACEHOLDER",
     "partyName": "ISSUE_TO_LEGAL_NAME_PLACEHOLDER",
-    "partyCodes": [
+    "sendToPlatform": "SEND_TO_PLATFORM_PLACEHOLDER",
+    "identifyingCodes": [
       {
-        "partyCode": "ISSUE_TO_PARTY_CODE_PLACEHOLDER",
         "codeListProvider": "EPUI",
```

Comment:
@palatsangeetha: The `EPUI` code is no longer valid for this field. I am a bit surprised that the CTK is giving green for this. I suspect one of the eBL conditional validations are not up to date (the same code is used for eBL, ISS, and PINT, so the bug is in all the standards).


@gj0dcsa: FYI in case you were planning to deploy it and someone test.

Reply:
> @palatsangeetha: The `EPUI` code is no longer valid for this field. I am a bit surprised that the CTK is giving green for this. I suspect one of the eBL conditional validations are not up to date (the same code is used for eBL, ISS, and PINT, so the bug is in all the standards).
> 
> @gj0dcsa: FYI in case you were planning to deploy it and someone test.

Thanks @nt-gt for pointing this out. I will update the request with example values from the latest API spec.